### ISSUE TITLE
NVDA log cannot be opened when logging is disabled

### ISSUE THREAD

**Issue by CyrilleB79** [nvaccess/nvda#19318]
### Steps to reproduce
1. Start NVDA normally
2. In General settings set log level to `DEBUG`
3. Use NVDA for a while
4. In General settings set log level to `Disabled`
5. Use NVDA for a while
6. Press `NVDA+f1` to open the log viewer
7. Open NVDA menu -> Tools -> View log

### Actual behavior
6. The log viewer is not opened and NVDA does not say anything
7. The log viewer is opened, with data logged until step 4.
### Expected behavior
At step 6 as at step 7, the log viewer should open with the content that has already been logged until step 4.

In case the log cannot be opened for any reason (e.g. empty log), at least NVDA should provide a speech / braille message feedback to the user.
### NVDA logs, crash dumps and other attachments
N/A
#### NVDA installed/portable/running from source
Installed
#### NVDA version
2025.3.2rc1
#### Windows version
Windows 11 23H2 (AMD64) build 22631.6199

But same issue with any version.
#### Name and version of other software in use when reproducing the issue
N/A
#### Other information about your system
N/A
#### Does the issue still occur after restarting your computer?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors
Issue already present on older NVDA versions
#### If NVDA add-ons are disabled, is your problem still occurring?
Yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes
